frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -107,20 +107,19 @@ function BookInfo:show(file, book_props, metadata_updated_caller_callback)
     end
     -- cover image
     local is_doc = self.document and true or false
-    if self.custom_book_cover == nil then
-        self.custom_book_cover = DocSettings:findCoverFile(file)
-    end
-    local callback = function()
+    self.custom_book_cover = DocSettings:findCoverFile(file)
+    local function showOrigCover()
         self:onShowBookCover(file, true)
     end
-    table.insert(kv_pairs, { _("Cover image:"), _("Tap to display"), callback=callback,
+    table.insert(kv_pairs, { _("Cover image:"), _("Tap to display"), callback=showOrigCover,
         separator=is_doc and not self.custom_book_cover })
```

Comment:
> Inline doesn't work.

Really? This would not work ?
```lua
table.insert(kv_pairs, {
    _("Cover image:"),
    _("Tap to display"),
    callback = function() self:onShowBookCover(file, true) end,
    separator = is_doc and not self.custom_book_cover,
})
```

Reply:
I tried without wrapping `function()`.